README.adoc

```diff
@@ -0,0 +1,203 @@
+----
+
+4. Run the application on the selected device
++
+[source, bash]
+----
+$ mtouch  --device=$DEVICE -launchsim ./Example.iOS/bin/iPhoneSimulator/Debug/Example.iOS.app
+----
+
+=== Run on Android emulator from command line
+
+To run the showcase app
+
+1. Add the Mono and Android tools executables to the path and setup the environment
++
+[source, bash]
+----
+$ export SDK_HOME=$HOME/Library/Android/sdk
+$ export PATH=$PATH:/Library/Frameworks/Xamarin.iOS.framework/Versions/Current/bin:$SDK_HOME/tools:$SDK_HOME/platform-tools
+----
+
+2. Compile the sources
++
+[source, bash]
+----
+$ nuget restore
+$ msbuild Example.Android/Example.Android.csproj /p:Configuration=Debug /p:EmbedAssembliesIntoApk=True /t:Clean,Rebuild,SignAndroidPackage
+----
+
+3. Get the launchable activity name
++
+[source, bash]
+----
+$ export AAPT=`find $SDK_HOME/build-tools/ -name aapt | sort -r | head -n 1`
+$ export ACTIVITY=`$AAPT dump badging Example.Android/bin/Debug/org.aerogear.xamarin.Example-Signed.apk | grep launchable-activity | perl -pe "s/.*name='(.*?)'.*/org.aerogear.xamarin.Example\/\1/"`
+----
+
+4. Start the android emulator
++
+[source, bash]
+----
+$ emulator -avd nexus_5x_api_27 -partition-size 512
+----
++
+The list of available avd can be obtained with
++
+[source, bash]
+----
+$ emulator -list-avds
+----
++
+If no avds are available, the easiest way to add one is to open Visual Studio and click on `Tools->Device Manager`.
+
+5. Install the Showcase app APK into the emulator and run it
++
+[source, bash]
+----
+$ adb -e install -r  ./Example.Android/bin/Debug/org.aerogear.xamarin.Example-Signed.apk
+$ adb shell am start -n $ACTIVITY
+----
+
+=== Work with Self-signed Certificate
+
+The easiest way to support self-signed certificate is to install the CA certificate that is used to sign the certificate onto the device/simulator.
+
+1. Get the CA certificate of the server. You can use this command:
++
+[source, bash]
+----
+openssl s_client -showcerts -connect host:port
+----
++
+It will print out the full certificate chain of the server and you should save the content of the root certificate into a PEM file.
+
+2. Install the CA certificate onto the device.
+    - To install onto an iOS simulator, you can simply drag & drop the certificate file over it.
+    - To install on an actual ios device, you can use a file sharing service or send the certificate file as an email attachment.
+    - To install onto an Android simulator:
+        a. Drag & drop the certificate file over it.
+        b. On the simulator, open the settings app
+        c. In the top bar (`Search settings`), write _Trusted credentials_
+        d. Click on `Trusted credentials`
+        e. Click on `Install from SD card`
+        f. Choose the file you just uploaded
+3. Ensure the newly installed CA certificate is trusted.
++
+On the iOS device/simulator, go to `Settings -> General -> About -> Certificate Trust Settings` and enable full trust for the certificate.
```

Comment:
link to this section instead: https://docs.aerogear.org/aerogear/latest/getting-started.html#using-self-signed-certificates-in-mobile-apps